content/blog/2024-10-04-guac_update.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "GUAC Update: October 2024"
+layout: post
+authors: 
+  - Ben Cotton
+date: 2024-10-04
+tags:
+  - guac-update
+  - community
+  - events
+  - releases
+---
+
+Welcome to the inaugural [GUAC Update](/tags/guac-update), a monthly review of what has happened in the GUAC community and what's coming up.
+If you have feedback, please [let us know]({{< relref "../community.md" >}}).
+To include something in next month's update, leave a comment in the [issue](https://github.com/guacsec/guac-landing/issues/97).
+
+## New contributors
+
+Thanks to the following people for making their first contributions in September
+(the people listed here may have contributed to other parts of the project previously):
+
+* [Anirudh Edpuganti](https://github.com/ANIRUDH-333): [guac-docs#149](https://github.com/guacsec/guac-docs/pull/149)
+* [Ben Cotton](https://github.com/funnelfiasco): several contributions to [GUAC Visualizer](https://github.com/guacsec/guac-visualizer)
+* [Parth Patel](https://github.com/pxp928): several contributions to [GUAC Visualizer](https://github.com/guacsec/guac-visualizer)
+
+## Releases
+
+We had several releases for GUAC and GUAC Visualizer in September.
+Enhancements and bug fixes abound.
+For GUAC, these releases include network retries for certifiers, improvements to SBOM parsing, and adding the ClearlyDefined certifier to the compose files.
+The GUAC Visualizer release adds a new information window that includes quick facts on vulnerabilities, SBOMs, and SLSA.
+
+Full details for each release are below:
+
+* [GUAC v0.8.3](https://github.com/guacsec/guac/releases/tag/v0.8.3)
+* [GUAC v0.8.4](https://github.com/guacsec/guac/releases/tag/v0.8.4) ([blog post]({{< relref "2024-09-11-guac-v0.8.4.md" >}}))
+* [GUAC v0.8.5](https://github.com/guacsec/guac/releases/tag/v0.8.5)
+* [GUAC v0.8.6](https://github.com/guacsec/guac/releases/tag/v0.8.6) ([blog post]({{< relref "2024-09-19-guac-v0.8.6.md" >}}))
+* [GUAC v0.8.7](https://github.com/guacsec/guac/releases/tag/v0.8.7) ([blog post]({{< relref "2024-09-25-guac-0.8.7.md" >}}))
+* [GUAC v0.8.8](https://github.com/guacsec/guac/releases/tag/v0.8.8)
```

Comment:
Can you add in the two releases after this and this should be set!

Reply:
Those were in October, so they'd be in the next post (but PR #99 adds a blog post for them)

Reply:
ah ok